lib/experimental/blocks.php

```diff
@@ -153,6 +154,12 @@ function gutenberg_render_block_connections( $block_content, $block, $block_inst
 				$attribute_value['value']
 			);
 
```

Comment:
The following will need some work. Currently, there is no common attribute that identifies what the outer wrapper tag is across blocks using innerBlocks. This currently just gets the first tag in the content, which works for lists, but may not for other blocks, so we may need to add an additional attribute to the blocks to clearly identify the innerBlocks wrapper.